ext/opentelemetry-ext-jaeger/README.rst

```diff
@@ -1,6 +1,12 @@
 OpenTelemetry Jaeger Exporter
 =============================
 
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-ext-jaeger.svg
+   :target: https://pypi.org/project/opentelemetry-ext-jaeger/
+
+
```

Comment:
Having a small description of the package would be great, similar to other readme files

Reply:
Done!